clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -82,7 +110,8 @@ export default async function handler(
       privacyCenterUrl: environment.settings.PRIVACY_CENTER_URL,
       fidesApiUrl: environment.settings.FIDES_API_URL,
     },
-    geolocation,
+    experience: experience || undefined,
```

Comment:
do we still need this call now that we have prefetching happening?
https://github.com/ethyca/fides/blob/9f081676c6426d6ce7a9d3be3f3adacd52e67df6/clients/fides-js/src/fides.ts#L245-L250

it seems like that would cause 2 calls to the server in the event that no experience exists (prefetch would call once, would say `experience: undefined`, and then fides-js would try to make the call again)

Reply:
`experience: undefined` would be returned in the case where for some reason a customer has overlay enabled but has not setup the necessary ENV vars nor notices to successfully obtain data from this endpoint. So then 2 calls to the server would be made, one from prefetch and one from `fides-js`.

Otherwise, the `fides-js` calls for geolocation and experience currently function as a safeguard in case the pre-fetch would fail for some reason. I'm a little torn whether to remove this now. @NevilleS any opinions here?

Reply:
Yeah, I don't think I see a way we could fully remove this, since we can't *guarantee* prefetching will work. So making the "standard" case be one where the JS client self-locates and fetches it's experience data makes sense, but then we support an "optimized" case where everything is prefetched and cached server-side to improve page-load times 👍 

As a relatively trivial counter-example: we can only prefetch if we can determine the geolocation server-side. We will never be able to *fully* guarantee that, so let's keep the client code (and tests for this case!) live forever